resources/migrations/001_update_migrations.yaml

```diff
@@ -8206,19 +8206,18 @@ databaseChangeLog:
               column:
                 name: dashboard_id
 
-  # last_used_at is used to determine if a report card is stale and should be cleaned up
-  # if the card is imported with serialization, we don't want to automatically clean up
+  # last_used_at will be used to determine if a report card is stale and should be cleaned up
+  # if the card is imported with serialization. We don't want to automatically clean up
   # newly imported cards, and we don't want to have to make it nullable.
+  # Because we weren't collecting this data before, we have to populate all existing cards
+  # with the current timestamp to provide a "lower bound" for when the card was last used.
   - changeSet:
-      id: v50.2024-07-09T20:04:01
+      id: v50.2024-07-09T20:04:00
```

Comment:
Changing this ID means that this will run again even if the previous changeset had run already, such as in stats.

Reply:
How do rollbacks work if it had been run with the old id, and now doesn't find the migration in the changeset

Reply:
They leave the migration id in the place and it stays there forever so you cannot downgrade your instance. So if it was present in release, we either have to delete the old migration ([like here](https://github.com/metabase/metabase/pull/44908))  or create a new migration (and dumb down the old one).

I also used that "hack" and then we've got somebody not being able to downgrade to 49. :)

Reply:
Wow, that is a bit painful. So they'd have to first migrate up to be able to migrate down?

Reply:
It seems like the easiest thing to do here is to just leave the old migration in place, and make this a new one. We'll run a potentially expensive query twice, but I think that's better than customers running into a rough edge.

Reply:
yeah, twiddling with migration IDs is a great way to end up in hard-to-fix states :/

Reply:
another option is to set `runOnChange: true` for this migration: https://docs.liquibase.com/concepts/changelogs/attributes/runonchange.html

this way existing instances will re-trigger this and we don't have to change the id here

Reply:
This hasn't been backported so I'm pretty sure it's fine. We can assume anyone running a jar from master isn't going to downgrade to 49. But TIL about `runOnChange: true`, I hadn't I've seen that before. I would probably stick with `validCheckSum: ...` if this was backported because I guess we agree that checksums are generally useful.

Reply:
> This hasn't been backported so I'm pretty sure it's fine.

It'll bring some pain to deploy on stats anyway, so be ready to handle that (similar rename by me was fixed by Ngoc last week).

Reply:
I'm pretty sure deploying this change on stats is fine because I'm changing the changeset's ID here. So stats will see this is as if one migration was deleted, and one migration was added, and run it as if it were a new migration. Changing a migration ID like this is only a problem on downgrades when liquibase looks for a rollback and there is none present.